RELEASE_NOTES.md

```diff
@@ -1,5 +1,31 @@
 # ChefDK 3.7 Release Notes
 
+## Chef 14.9.13
+
+ChefDK now ships with Chef 14.9.13. See <https://docs.chef.io/release_notes.html> for more information on what's new.
+
+## InSpec 3.3.14
+
+ChefDK now ships with Inspec 3.3.14. See <https://github.com/inspec/inspec/releases/tag/v3.3.14> for more information on what's new.
+
+## Updated Components and Tools
+
+### kitchen-inspec 1.0.1
+
+Support bastion configuration in transport options
+
+### Other updates
+
+* `bundler`: 1.16.1 -> 1.17.3
+* `chef-apply`: 0.2.4 -> 0.2.7
+* `chefstyle`: 0.11.0 -> 0.12.0
+* `cookstyle`: 3.0.2 -> 4.0.0
+* `kitchen-azurerm`: 0.14.7 -> 0.14.8
+* `kitchen-vagrant`: 1.3.6 -> 1.4.0
+* `kitchen-tidy`: 1.2.0 -> 2.0.0
+* `rubocop`: 0.55.0 -> 0.62.0
```

Comment:
This is part of the cookstyle update and shouldn't go out

Reply:
fixed.